schema.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Why do we need this file, where is the test cases ?

Reply:
Do you mean that we shouldn't push the schema?
And for the test cases, we made a new file called ```test_create```

Reply:
You didn’t push the test case when u push this file. Now it is Find